app/src/main/java/net/sourceforge/opencamera/MainActivity.java

```diff
@@ -741,6 +743,10 @@ public void onInit(int status) {
             notificationManager.createNotificationChannel(channel);
         }
 
+        // create notification channel to uploading files
+        applicationInterface.getFileUploadService().createUploadServiceNotificationChannel();
+        UploadServiceConfig.initialize(getApplication(), "UploadChannel", BuildConfig.DEBUG);
```

Comment:
I think it should be done not here! As it is mentioned in docs of the library "init in the app class" --- so in extendedApp in our case

Reply:
The docs says about Application, I think it's not worth writing this in OpenCameraApplication. In the interface, I don't see much point in doing this either. I will make it in a new class and pass the Application there